integration/shareddata/benchmark.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package shareddata provides data for tests and benchmarks.
+package shareddata
+
+import "go.mongodb.org/mongo-driver/bson"
+
+// SimpleBenchmarkValues contain documents with various types of values.
+var SimpleBenchmarkValues BenchmarkValues = NewBenchmarkValues(
+	"Ha59TmC9nwTXqEob73HIwqWUcx2Lp7aengCtNsqjWK0=",
+	GenerateMixedValues(),
+)
+
```

Comment:
To make sure that generator produce exact same data every time in the same order, we calculate the checksum of all inserted documents to be sure that expected and actual checksum are the same.

Checksum is also used in benchmark test names. It's good for tooling like benchstat as after changing BenchmarkValues data, the results will not be compared with the old values.